clients/python/girder_client/cli.py

```diff
@@ -53,10 +53,10 @@ def formatSize(length):
                 unit = ''
                 units = ['K', 'M', 'G', 'P']
```

Comment:
Am I missing something here, or shouldn't there be a `'T'` between `'G'` and `'P'`?